src/app/public/modules/flyout/flyout-adapter.service.ts

```diff
@@ -48,4 +56,23 @@ export class SkyFlyoutAdapterService {
       iframes[i].style.pointerEvents = enable ? '' : 'none';
     }
   }
+
+  public shouldClickCloseFlyout(host: ComponentRef<SkyFlyoutComponent>, target: Node): boolean {
```

Comment:
This method does the job really well, but I found it difficult to parse out what it was doing. Would you mind providing some well-written comments to describe the logic here? This will help us in the future when we have to come back to the code.

Reply:
Fixed